src/css-grids/cssgrids.html

```diff
@@ -0,0 +1,151 @@
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="./cssgridsStyles.css">
+    <link rel="stylesheet" href="./reset.css">
+    <title>Widget Dashboard</title>
+</head>
+
+    <body> 
+    <!-- will hold total grid -->
+        <section id="clock">
+            <!--stolen clock code-->
+            <input type="time" id="clock" value="12:12:12">
+            <script>
+                let time = {
+                hours: 24,
+                minutes: 59,
+                seconds: 58 
+                }
+                let t;
+                let custom_input = document.getElementById("clock");
+
+                custom_input.addEventListener("focus", stopTime);
+                custom_input.addEventListener("focusout", startTime);
+
+                function stopTime() {
+                clearTimeout(t);
+                }
+
+                function startTime() {
+                let arrTime = custom_input.value.split(':');
+                
+                time.hours = arrTime[0];
+                time.minutes = arrTime[1];
+                time.seconds = arrTime[2];
+                
+                clock();
+                }
+
+                function checkSeconds(seconds) {
+                let parsedSeconds = Number(seconds);
+                
+                if (parsedSeconds >= 60) {
+                    time.seconds = "0" + 1;
+                    
+                    time.minutes++;
+                    
+                } else if (parsedSeconds < 10) {
+                    time.seconds = "0" + parsedSeconds;
+                }
+                
+                checkMinutes(time.minutes)
+                }
+
+                function checkMinutes(minutes) {
+                let parsedMinutes = Number(minutes);
+                
+                if (parsedMinutes >= 60) {
+                    time.minutes = "00";
+                    time.hours++;
+                } else if (parsedMinutes < 10) {
+                    time.minutes = "0" + parsedMinutes;
+                }
+                
+                checkHours(time.hours)
+                }
+
+                function checkHours(hours) {
+                let parsedHours = Number(hours);
+
+                if (parsedHours > 24) {
+                    time.hours = "00"
+                } else if (parsedHours < 10) {
+                    time.hours = "0" + parsedHours;
+                }
+                }
+
+                function clock() {
+                time.seconds++;
+
+                checkSeconds(time.seconds);
+
+                document.getElementById("clock").innerHTML = (time.hours + ":" + time.minutes + ":" + time.seconds);
+                
+                custom_input.value = time.hours + ":" + time.minutes + ":" + time.seconds;
+                
+                t = setTimeout('clock()', 1000); 
+                }
```

Comment:
i like that you're getting your feet wet with javascript but don't worry too much about it now! that will come. lets focus on html/css first!